04/app.js

```diff
@@ -1 +1,52 @@
-console.log('DOM');
\ No newline at end of file
+console.log('DOM');
+
+/*const ulCreate = document.createElement('ul')
+console.log(ulCreate)
+const liFirstCreate = document.createElement('li')
+//liFirstCreate.innerText = 'home'
+const aFirstCreate = document.createElement('a')
+//console.log(liFirstCreate)
+ulCreate.appendChild(liFirstCreate)
+liFirstCreate.appendChild(aFirstCreate)
+document.querySelector('nav').appendChild(ulCreate)
+aFirstCreate.setAttribute('href', '/')
+aFirstCreate.innerText = 'home'
+
+const liSecondCreate = document.createElement('li')
+const aSecondCreate = document.createElement('a')
+ulCreate.appendChild(liSecondCreate)
+liSecondCreate.appendChild(aSecondCreate)
+aSecondCreate.setAttribute('href', '/gallery')
+aSecondCreate.innerText = 'gallery'
+
+const liThirdCreate = document.createElement('li')
+const aThirdCreate = document.createElement('a')
+ulCreate.appendChild(liThirdCreate)
+liThirdCreate.appendChild(aThirdCreate)
+aThirdCreate.setAttribute('href', '/contact')
+aThirdCreate.innerText = 'contact'*/
+
+const ulCreate = document.createElement('ul');
+document.querySelector('nav').appendChild(ulCreate);
+console.log(ulCreate);
+
+
+const menuItems = [
+    {text: 'start', url: '/'},
+    {text: 'galeria', url: '/gallery'},
+    {text: 'kontakt', url: '/contact'},
+]
+menuItems.forEach(function(el)
+{
+    if(ulCreate){
```

Comment:
ten if nie jest niezbędny, sam stworzyłeś `ul` więc masz pewność że on tam jest :)
Jak już bardzo chciałbyś zrobić if-a to lepiej go wstawić przed  `forEach`.
Zwróć uwagę, że wew. pętli sprawdzasz za każdym razem czy istnieje,
a przecież możesz to zrobić jeden raz i efekt będzie ten sam.